wrapping/itkImageToImageFilter.wrap

```diff
@@ -0,0 +1,27 @@
+# Extra wrapping definitions here are needed for
+# Spectra1DAveragingImageFilter and Spectra1DNormalizeImageFilter
```

Comment:
I believe there are also wrappings for `ImageToImageFilter` in `CurvilinearArraySpecialCoordinatesImage.wrap`, should those be moved under this file as well?

Reply:
Yes, sure. That should be done in a separate PR, later.